### DISCUSSION TITLE
Need some help getting rest of Renovate PRs merged in

### DISCUSSION THREAD

**Discussion by hazendaz** [spotbugs/spotbugs#2396] [General]
I'm a windows user, the spotbugs build as it is doesn't work in windows properly.  I've tried to address and work towards resolving but not a lot of time myself to deal with and its not really in my want to do list of priorities.  That said, we have 4 or so PRs that are blocked that seem should be easy to get merged.  If anyone could take a bit of time to address especially the bcel one that would be great!

**Comment by hazendaz** [comment]
Also noting, I'm not a gradle user either which makes it harder for me to figure out ;)

**Comment by JuditKnoll** [comment]
The PRs this discussion is about, already got merged, so I'm closing this.
